tests/dubbo-java/README.md

```diff
@@ -15,3 +15,10 @@ mvn -Djava.net.preferIPv4Stack=true -Dexec.mainClass=org.apache.dubbo.tests.clie
 mvn clean package
 mvn -Djava.net.preferIPv4Stack=true -Dexec.mainClass=org.apache.dubbo.tests.provider.Application exec:java
 ```
+
+## Start the service provider
```

Comment:
`with registry`

Reply:
Maybe it's better to make it a command line arguement to control whether it should register itself into the zookeeper? This saves us a new Application and a new provider.

Reply:
Ack. I would fix this.